pkg/profile/delete.go

```diff
@@ -20,3 +22,15 @@ func Delete(
 	}
 	return nil
 }
+
+func Delete(cli client.Client, ns, profileName string) error {
+	ctx := context.Background()
+	prof := v1.Profile{
+		TypeMeta: metav1.TypeMeta{},
```

Comment:
Just curious: does this work? Not specifying full GVR / type info? If you tested it and it's fine, than that's ok.

Reply:
Yes, we did test it, please check this comment: https://github.com/arlonproj/arlon/pull/196#discussion_r962577023